### DISCUSSION TITLE
./gradlew -PappVerName=${{ env.VERSION }} 好像不生效

### DISCUSSION THREAD

**Discussion by Doddddd** [LibChecker/LibChecker#791] [Q&A] [answered]
https://github.com/LibChecker/LibChecker/blob/0ccf927705733943272eff9a4888024be69f8318/.github/workflows/android.yml#LL77C47-L77C47

**Comment by zhaobozhen** [comment, marked as answer]
这个参数应该已经没用了，可以忽略
